### DISCUSSION TITLE
Convert Epoch Time to Date Time

### DISCUSSION THREAD

**Discussion by Sandy381** [fbaligand/kibana-enhanced-table#308] [General]
Hi,

I'm having a field with EPOCH times in string format in Enhanced Table

So I'm trying to convert EPOCH values to Date time in Enhanced Table.

First , I create a calculated column with EPOCH values in Number values.

Then, I try to convert the EPOCH to Date time in my format DD-MMM-YYYY format.

But, I'm not able to convert using parseDate

Below Screenshot of the issue
![Screenshot 2023-02-09 at 10 16 50 AM](https://user-images.githubusercontent.com/74658016/217721484-d3057b9d-955d-4661-b742-f4e073981334.png)

Pls Help.

**Comment by fbaligand** [comment]
First, you don't need to use `parseDate` function, given that it takes a ISO-8601 formatted date and converts it to the number of milliseconds since Epoch.
Then, expected value for a "Date" formatted column is the number of **milliseconds** since Epoch.
It seems that your "Execution Date" is in seconds.
So I invite you to use this formula: `col2 * 1000`

**Reply by Sandy381** [reply]
Yep, That was the issue.

The time was in seconds, now I multiplied by 1000 & converted to millisec and I'm able to see in Time Date format!.

Thanks!

**Reply by fbaligand** [reply]
Great!
